### PR TITLE
Improve Rector success message.

### LINKED ISSUE #2606: Change output message when running in dry-run
> Rector is done! 1 changed files

When running with `--dry-run`, it should say something like "1 files would have been changed".

### PR DESCRIPTION
**Pull request by gnutix**
Fixes #2606. Makes for nicer human-readable messages. I didn't touch the json format output, as I don't think it should change. Let me know if you feel otherwise.

### CODE REVIEW

File: src/Console/Output/ConsoleOutputFormatter.php [hunk 1]
```diff
@@ -47,10 +48,18 @@ public function report(ErrorAndDiffCollector $errorAndDiffCollector): void
             return;
         }
 
-        $this->symfonyStyle->success(sprintf(
-            'Rector is done! %d changed files',
-            count($errorAndDiffCollector->getFileDiffs()) + $errorAndDiffCollector->getRemovedAndAddedFilesCount()
-        ));
+        $changes = $errorAndDiffCollector->getFileDiffsCount() + $errorAndDiffCollector->getRemovedAndAddedFilesCount();
```
Comment (TomasVotruba) [inline src/Console/Output/ConsoleOutputFormatter.php]:
```suggestion
-       $changeCount = $errorAndDiffCollector->getFileDiffsCount() + $errorAndDiffCollector->getRemovedAndAddedFilesCount();
```

Comment (TomasVotruba) [inline src/Console/Output/ConsoleOutputFormatter.php]:
```suggestion
        $changeCount = $errorAndDiffCollector->getFileDiffsCount() + $errorAndDiffCollector->getRemovedAndAddedFilesCount();
```

File: src/Console/Output/ConsoleOutputFormatter.php [hunk 2]
```diff
@@ -37,7 +38,7 @@ public function __construct(SymfonyStyle $symfonyStyle, BetterStandardPrinter $b
         $this->betterStandardPrinter = $betterStandardPrinter;
     }
 
-    public function report(ErrorAndDiffCollector $errorAndDiffCollector): void
+    public function report(ErrorAndDiffCollector $errorAndDiffCollector, Configuration $configuration): void
```
Comment (TomasVotruba) [inline src/Console/Output/ConsoleOutputFormatter.php]: Why passing service as a method argument?
Should be in constructor if needed.

Comment (gnutix) [reply src/Console/Output/ConsoleOutputFormatter.php]: Oh right, it is a service indeed. At first I thought it was a value object. I'll change it.

Comment (TomasVotruba) [reply src/Console/Output/ConsoleOutputFormatter.php]: I see. Could you think of better name for it so it's recognizable?

Comment (gnutix) [reply src/Console/Output/ConsoleOutputFormatter.php]: I don't think the name is wrong or misleading. Those were just my expectations of Symfony Command input arguments handling. ;)

**Comment by TomasVotruba** [comment]
Thank you :+1:

**Pull request merged**